api/web3server_marshal.go

```diff
@@ -96,11 +97,11 @@ type (
 		ErrorString   string                      `json:"error"`
 	}
 	debugTraceTransactionResult struct {
-		Failed      bool        `json:"failed"`
-		Revert      string      `json:"revert"`
-		ReturnValue string      `json:"returnValue"`
-		Gas         uint64      `json:"gas"`
-		StructLogs  []structLog `json:"structLogs"`
```

Comment:
should delete the definition of `structLog` at L83 above? since it is moved to api/types/traces.go?